etc/repl/user.clj

```diff
@@ -0,0 +1,10 @@
+(ns user
+  (:require [figwheel-sidecar.system :as sys]
+            [clojure.pprint :refer [pprint]]
+            [com.stuartsierra.component :as component]))
+
+(def system (component/system-map :figwheel-system (sys/figwheel-system (sys/fetch-config))))
+
+(defn go! []
+  (alter-var-root #'system component/start)
+  (sys/cljs-repl (:figwheel-system system)))
```

Comment:
I think there's a repl file in the `script` directory. Would be good to keep it as one.

Reply:
I wish I had asked